### ISSUE TITLE
Key bind(`ctrl + U`) doesn't work correctly.

### ISSUE THREAD

**Issue by obregonia1** [ruby/irb#324]
## Description

`ctrl + U` should work delete from the cursor to the beginning of the line but it works delete from the cursor to the end of the line - like `crtl + K`.

As running with `--legacy` option, it works correctly.

## Result of irb_info

```
irb(main):001:0> irb_info
=> 
Ruby version: 3.1.0
IRB version: irb 1.4.1 (2021-12-25)
InputMethod: ReidlineInputMethod with Reline 0.3.0
.irbrc path: /Users/kentaro/.irbrc
```

## Terminal Emulator

Terminal.app

## Setting Files

Are you using `~/.irbrc` and `~/.inputrc`?

No

**Comment by obregonia1** [comment]
This problem was caused by reline.

**Comment by dechimp** [comment]
sad.  `ctrl-w`  is the best workaround I've found.

**Comment by mentalisttraceur** [comment]
For context, the reline bug is https://github.com/ruby/reline/issues/415

**Comment by ima1zumi** [comment]
This problem has been fixed since Reline v0.3.1. https://github.com/ruby/reline/commit/3ddda1d982000f89248aafd202fc2a34ae73c0da
Please upgrade your version of Reline. If the problem persists, please report it.

**Comment by mentalisttraceur** [comment]
@ima1zumi Thanks, but note that I also linked to the same fix (indirectly, through the Reline issue, which links to the PR with the fix).

I just commented so that others like me who find this first know it was already fixed and have a direct link to the Reline issue, so they don't have to go looking for the information like I did.
